### DISCUSSION TITLE
After migration from Forestry unable to find collections

### DISCUSSION THREAD

**Discussion by BorisMolch** [tinacms/tinacms#4234] [Help]
seeing error Unable to find collection for file at data\en\homepage.yml
for most of the collectoins (not all)

not sure if i should edit the config.ts (tried to change to correct format)
im on windows10
hugo

+-- @tinacms/cli@1.5.30
| +-- @tinacms/app@1.2.28
| | -- tinacms@1.5.21 deduped
| -- tinacms@1.5.21 deduped
`-- tinacms@1.5.21

**Comment by BorisMolch** [comment]
oh i see it might be same as https://github.com/tinacms/tinacms/issues/4210

**Reply by BorisMolch** [reply]
After changing the ` format`,` frontmatterFormat` and `frontmatterDelimiters`
the error no longer shows. 

but the collections are empty 
during the migration i saw these errors:

No fields found for data/en/homepage.yml. Please add the fields you want to edit to the Homepage collection in the config file.
No fields found for data/en/blog.yml. Please add the fields you want to edit to the Blog settings collection in the config file.
Info: Match _index.md was transformed to _index. See https://tina.io/docs/forestry/common-errors/#info-match-match-was-transformed-to-newmatch
No fields found for data/en/community.yml. Please add the fields you want to edit to the Community Info collection in the config file.
Info: Match _index.md was transformed to _index. See https://tina.io/docs/forestry/common-errors/#info-match-match-was-transformed-to-newmatch
No fields found for data/en/patchathon.yml. Please add the fields you want to edit to the Patchathon collection in the config file.
No fields found for data/en/livepatchathon.yml. Please add the fields you want to edit to the Patchathon Live Stream collection in the config file.
No fields found for data/en/about.yml. Please add the fields you want to edit to the About collection in the config file.
Info: Match _index.md was transformed to _index. See https://tina.io/docs/forestry/common-errors/#info-match-match-was-transformed-to-newmatch
No fields found for data/en/team.yml. Please add the fields you want to edit to the Team collection in the config file.
No templates found for section Jobs. Please see https://tina.io/docs/forestry/content-modelling/ for more information
No fields found for data/en/faq.yml. Please add the fields you want to edit to the FAQ collection in the config file.
No fields found for data/en/japanese.yml. Please add the fields you want to edit to the Japanese collection in the config file.
No fields found for data/en/links.yml. Please add the fields you want to edit to the Linktree collection in the config file.
No fields found for i18n/en.yaml. Please add the fields you want to edit to the i18n Strings collection in the config file.

**Comment by kldavis4** [comment]
> oh i see it might be same as #4210

Yes, it is probably that. You can try adjusting the match properties until we can get that bug fixed.

**Comment by BorisMolch** [comment]
> > oh i see it might be same as #4210
> 
> Yes, it is probably that. You can try adjusting the match properties until we can get that bug fixed.

im not sure if its the same, becuase the "unable to find collection" error i could manage to resolve with correct format.
but the error on migration, not being able to find the fields... looks like a different issue.

**Reply by kldavis4** [reply]
is your repository public or can you invite my github user and I can take a look?

**Reply by BorisMolch** [reply]
i made a copy of the repo @kldavis4 invited you too
https://github.com/BorisMolch/pfrontend-temp

**Reply by kldavis4** [reply]
Thanks. I'll take a look next week

**Reply by BorisMolch** [reply]
thank you!

**Reply by kldavis4** [reply]
For the homepage collection, I was able to get it working by changing `format: "yaml"` to `format: "yml"`. `frontmatterFormat` and `frontmatterDelimiters` is not required for the yml collections

**Reply by BorisMolch** [reply]
@kldavis4 When i try this i get` [Error] GetDocument failed: Unable to fetch, errors: Unable to find collection for file at data\en\homepage.yml`

**Reply by kldavis4** [reply]
yes, I suspect that is #4210 since I am testing on a mac and not seeing the issue

**Reply by kldavis4** [reply]
opened a pr that should fix the windows issue: #4245

**Comment by BorisMolch** [comment]
Ok thank you ! Will try on a Mac machine

**Comment by BorisMolch** [comment]
> For the homepage collection, I was able to get it working by changing `format: "yaml"` to `format: "yml"`. `frontmatterFormat` and `frontmatterDelimiters` is not required for the yml collections

were you able to see documents in homepage?
i repeated steps of migration on a mac.
There are no errors but homepage collection (and most of others) have no documents in collection when opening tina admin.

**Reply by kldavis4** [reply]
Yes - see this pull request: https://github.com/BorisMolch/pfrontend-temp/pull/1

**Reply by BorisMolch** [reply]
ah ok. yes i see , those new fields i added manually do show.
but none of the old structure is showing.
for example "blog settings" @kldavis4

**Reply by kldavis4** [reply]
did you make the corresponding change to that collection (like a made to homepage)? Please push up your updates so I can see what you are working with.

**Reply by BorisMolch** [reply]
@kldavis4 I force pushed the current state to the duplicate repo https://github.com/BorisMolch/pfrontend-temp
currently:
- Hompage and many others do not have any documents or items in collection. 
- Locally when i navigate to our webpage/admin it opens in local mode. Trying on another machine gives error 
```
Failed loading TinaCMS assets
Your TinaCMS configuration may be misconfigured, and we could not load the assets for this page.

Please visit [this doc](https://tina.io/docs/tina-cloud/faq/#how-do-i-resolve-failed-loading-tinacms-assets-error) for help.
```
steps in doc seem to be what i've done already

**Reply by BorisMolch** [reply]
on the original repo deploy action was seemingly successful
<img width="734" alt="image" src="https://github.com/tinacms/tinacms/assets/33655927/04703f87-7799-497b-a75c-5dc11356b0e8">

**Reply by kldavis4** [reply]
I ran your site locally after the update and I am seeing a homepage document with a dummy field, which corresponds to what is in the tina schema: https://github.com/BorisMolch/pfrontend-temp/blob/master/tina/config.js#L41 - not sure what happened to the fields that were previously defined: https://github.com/BorisMolch/pfrontend-temp/blob/kldavis4/tina/config.ts#L44

**Reply by BorisMolch** [reply]
- The fields previously defined were just a test.
Im looking for the actual content of the website. Seems like it was not migrated, and needs to be defined manually. 
- After deploying the site. The admin is not accessible 
`Failed loading TinaCMS assets` (as mentioned above)
this is another issue, should i open another git issue for it ?
Thank you @kldavis4

**Reply by kldavis4** [reply]
if you haven't already, check out: https://tina.io/docs/tina-cloud/faq/#how-do-i-resolve-failed-loading-tinacms-assets-error

If you aren't able to resolve it, please use our discord: https://discord.com/invite/zumN63Ybpf (post in the #help channel)
